environment.yml

```diff
@@ -72,3 +73,6 @@ dependencies:
   # Not on conda forge - vprof
   - yamllint
   - yapf
+  - pip
+  - pip:
+      - ESMValTool_sample_data==0.0.3
```

Comment:
aghh I really don't like this - can we not make it a conda forge package?

Reply:
It's only for testing purposes, I'm not sure if we should bother the busy conda-forge peeps with it.

Reply:
yeah but this type of mixed conda-pip conda environment file is a bit of a no-no, can we not have it in `setup.py` and post-install it with `pip`? This will not get into the conda package anyway since you're not allowed mixed conda-pip stuff in the conda recipe

Reply:
Why is that better?

Reply:
plus, it falls in the same category with vprof, conda renegade

Reply:
Yeah, `vprof`, that's something else to think about (is it still up to date and the best choice?). Anyway, I'll remove those lines again if they bother you, they're quite irrelevant anyway.